<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm Jock</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
    <item>
     <widget class="DigitalClock" name="lcdClock">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hboxSpins">
      <item>
       <widget class="QSpinBox" name="spnHour">
        <property name="maximum">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnMinute">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnSecond">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hboxButtons">
      <item>
       <widget class="QPushButton" name="btnTest">
        <property name="text">
         <string>Test Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save Alarm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QLCDNumber</extends>
   <header>digitalclock.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
